calm/draft/2024-10/meta/flow.json

```
@@ -0,0 +1,84 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-09/meta/flow.json",
+  "title": "Business Flow Model",
+  "description": "Defines business flows that relate to technical architectures, allowing mapping of flows to technical components and attaching control requirements.",
+  "defs": {
+    "flow": {
+      "type": "object",
+      "properties": {
+        "flowUniqueId": {
+          "type": "string",
+          "description": "Unique identifier for the flow"
+        },
+        "name": {
+          "type": "string",
+          "description": "Descriptive name for the business flow"
+        },
+        "description": {
+          "type": "string",
+          "description": "Detailed description of the flow's purpose"
+        },
+        "flow-requirement-url": {
+          "type": "string",
+          "description": "The requirement schema that specifies how this flow should be defined",
+        },
+        "flow-requirement-instance": {
+          "type": "string",
+          "description": "The URL of the specific requirement instance that implements the generic requirement. This should point to an instance specific to this flow."
+        },
+        "transitions": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "relationshipUniqueId": {
+                "type": "string",
+                "description": "Unique identifier for the relationship in the architecture"
+              },
+              "sequenceNumber": {
+                "type": "integer",
+                "description": "Indicates the sequence of the relationship in the flow"
+              }
+            },
+            "required": [
+              "relationshipUniqueId",
+              "sequenceNumber"
+            ]
+          }
+        },
+        "controlRequirements": {
+          "type": "array",
+          "items": {
+            "$ref": "https://calm.finos.org/draft/2024-09/meta/control-requirement.json#"
+          }
+        },
+        "metadata": {
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        }
+      },
+      "required": [
+        "flowUniqueId",
+        "name",
+        "description",
+        "transitions",
+        "flow-requirement-url"
+      ]
+    },
+    "flows": {
+      "type": "object",
+      "properties": {
+        "flows": {
+          "type": "array",
+          "items": {
+            "$ref": "#/defs/flow"
+          }
+        }
+      }
+    }
```

You have an object with an embedded array with the same name, we should pull the array up to the top level.

```suggestion
    "flows": {
      "type": "array",
        "items": {
          "$ref": "#/defs/flow"
        }
    }
```